### DISCUSSION TITLE
Pass ICY metadata (e.g. current song) from a stream through as a player attribute

### DISCUSSION THREAD

**Discussion by bennnnnnnn** [music-assistant/support#942] [Feature requests and ideas]
Hi,

Logging this as a reminder from the question I posed in the Discord channel.

Relevant reply from Marcel:
https://discord.com/channels/753947050995089438/977201339002142760/1029687563289641060

Kind regards,
Ben

**Comment by OzGav** [comment]
MA version 1 request.
